### ISSUE TITLE
Make the templates customizable.

### ISSUE THREAD

**Issue by ecraven** [academic-moodle-cooperation/moodle-atto_bsgrid#7]
Add an interface to edit templates and add your own new ones.

**Comment by gthomas2** [comment]
Custom templates will create issues for localisation.
Would it be better to have a separate plugin (based on this one) for custom bootstrap layouts?

**Comment by ecraven** [comment]
> Custom templates will create issues for localisation.
> Would it be better to have a separate plugin (based on this one) for custom bootstrap layouts?

Yes, the thing I'd like to do is move the templates into the php code,
so that you don't have to edit two pieces (the javascript and the php
settings page) when you add new templates.

Greetings, Peter
